<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cvToolsClass</class>
 <widget class="QWidget" name="cvToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cvTools</string>
  </property>
  <widget class="QLabel" name="InputImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/cvTools/image/R.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="OutputImg">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>100</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>图片操作</string>
   </property>
   <widget class="QPushButton" name="PB_OPENIMG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_RESIZEIMG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_CVTCOLORIMG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_HIST">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>直方图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_EnhanceImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>增强对比度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_OTSU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OTSU算法</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>二值化</string>
   </property>
   <widget class="QSlider" name="S_THRESHOLD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_THRESHOLD">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>腐蚀滤波</string>
   </property>
   <widget class="QSlider" name="S_ERODE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_ERODE">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>700</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>膨胀滤波</string>
   </property>
   <widget class="QSlider" name="S_DILATE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_DILATE">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox4_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>790</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>开操作</string>
   </property>
   <widget class="QSlider" name="S_OPEN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_OPEN">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox4_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>880</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>闭操作</string>
   </property>
   <widget class="QSlider" name="S_CLOSE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_CLOSE">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>均值滤波</string>
   </property>
   <widget class="QSlider" name="S_BLUR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_BLUR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>610</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>高斯滤波</string>
   </property>
   <widget class="QSlider" name="S_GAUSSIANBLUR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_GAUSSIANBLUR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>700</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>中值滤波</string>
   </property>
   <widget class="QSlider" name="S_MEDIANBLUR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MEDIANBLUR">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>790</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>双边滤波</string>
   </property>
   <widget class="QSlider" name="S_bilateralFilter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_bilateralFilter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>880</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>局部均值滤波</string>
   </property>
   <widget class="QSlider" name="S_BOXFILTER">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_BOXFILTER">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>边缘检测</string>
   </property>
   <widget class="QSlider" name="S_CANNYLOW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LOW</string>
    </property>
   </widget>
   <widget class="QSlider" name="S_CANNYHIGH">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HIGH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_CANNYLOW">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_CANNYHIGH">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>93</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>----</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>650</y>
     <width>120</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>对比度算法</string>
   </property>
   <widget class="QPushButton" name="PB_db1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>on/off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_db2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>off/on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_db3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>off-on/off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_db4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>on/off-on</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>810</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>对比度</string>
   </property>
   <widget class="QSlider" name="S_Contrast">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>880</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gamma</string>
   </property>
   <widget class="QSlider" name="S_Gamma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>拼图</string>
   </property>
   <widget class="QPushButton" name="PB_NewMat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始拼图</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Number">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>650</y>
     <width>111</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_BandPassUp">
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="PB_ChangToColorImg">
      <property name="text">
       <string>伪彩图</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="PB_ReadTiff">
      <property name="text">
       <string>读取tiff</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LE_ColorImgDown">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>up</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_ColorImgUp">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="PB_BandPass">
      <property name="text">
       <string>BandPassFFT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LE_BandPassDown">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>down</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>up</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ColorCard_1">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>16</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>图片叠加</string>
   </property>
   <widget class="QPushButton" name="PB_Add2Pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>叠加两张图片</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cvTools.qrc"/>
 </resources>
 <connections/>
</ui>
